Capfile

```diff
@@ -1,4 +1,12 @@
-load 'deploy'
-# Uncomment if you are using Rails' asset pipeline
-    # load 'deploy/assets'
-load 'config/deploy' # remove this line to skip loading any of the default tasks
\ No newline at end of file
+# Load DSL and Setup Up Stages
+require 'capistrano/setup'
+
+# Includes default deployment tasks
+require 'capistrano/deploy'
+
+# Run Composer and NPM install tasks
+require 'capistrano/composer'
+require 'capistrano/npm'
+
+# Loads custom tasks from `lib/capistrano/tasks' if you have any defined.
+Dir.glob('lib/capistrano/tasks/*.rake').each { |r| import r }
```

Comment:
This is nitpicky, but does it make more sense to put these in `config/capistrano` or something like that? Seems kinda odd to have a three-level deep folder structure just for this one rake file.